src/plugins/intel_cpu/src/nodes/kernels/x64/rms_kernel.hpp

```diff
@@ -0,0 +1,88 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "jit_kernel_base.hpp"
+
+#if defined(OPENVINO_ARCH_X86_64)
```

Comment:
We are compiling x64 folder only on x64 devices. In other words it is better to aboid OPENVINO_ARCH_X86_64 macroses inside such files, but rather do conditional include of `rms_kernel.hpp`

Reply:
Done, the macro is removed from the file.